src/catacomb.rs

```diff
@@ -129,6 +133,13 @@ impl<B: Backend + 'static> Catacomb<B> {
         // XDG output protocol.
         xdg::init_xdg_output_manager(&mut display, None);
 
+        // Run user startup script.
+        if let Some(mut home) = dirs::config_dir() {
+            home.push(Path::new("catacomb"));
+            home.push(Path::new(POST_START_SCRIPT));
```

Comment:
Push seems to take an `AsRef<Path>` as argument, so you can just push without the `Path::new`.